tools/souper-check.cpp

```diff
@@ -81,6 +81,7 @@ int SolveInst(const MemoryBufferRef &MB, Solver *S) {
         PrintReplacement(llvm::outs(), Rep.BPCs, Rep.PCs, Rep.Mapping);
       } else if (PrintReplSplit) {
         ReplacementContext Context;
+        Rep.Mapping.LHS->DemandedBitsVal = APInt::getAllOnesValue(Rep.Mapping.LHS->Width);
```

Comment:
I don't think these can be right -- this field should get initialized elsewhere, like all of the other fields